keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -134,6 +141,10 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
         |artifacts:
         |- name: keel
         |  type: deb
+        |  vmOptions:
+        |    baseOs: bionic
+        |    regions:
+        |    - ap-south-1
         |environments:
         |- name: test
         |  resources:
```

Comment:
Example of the new config in use. Note that `baseLabel` and `storeType` seem to me to have sensible defaults so I've omitted them.